explainer.md

```diff
@@ -297,7 +297,7 @@ function endVRSession() {
 function onSessionEnd() {
   vrSession = null;
 
-  // Ending the session stops executing callbacks passed to requestFrame().
+  // Ending the session stops executing callbacks passed to requestAnimationFrame().
```

Comment:
Now that the function names are the same, this is a bit confusing.  Perhaps prefix with VRSession?